scipy/stats/_stats_py.py

```diff
@@ -7166,29 +7172,35 @@ def warn_masked(arg):
     return Power_divergenceResult(stat, pvalue)
 
 
-def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
-    """Calculate a one-way chi-square test.
+def chisquare(f_obs, f_exp=None, ddof=0, axis=0, *, sum_check=True):
+    """Perform Pearson's chi-squared test.
 
-    The chi-square test tests the null hypothesis that the categorical data
-    has the given frequencies.
+    Pearson's chi-squared test [1]_ assesses the null hypothesis that the observed
+    frequencies (counts) are a sample from a multinomial distribution; that is,
+    they are the counts after *N* total observations are taken from a categorical
+    distribution.
```

Comment:
```suggestion
    Pearson's chi-squared test [1]_ is a goodness-of-fit test for a multinomial
    distribution with given probabilities; that is, it assesses the null hypothesis
    that the observed  frequencies (counts) are obtained by independent
    sampling of *N* observations with the given expected frequencies.
```

I'm not sure if that's better - in particular, I'm trying to make more clear that the test isn't just asking whether they follow *any* multinomial, but rather the particular multinomial with `f_exp` probabilities.

Reply:
I think it is good to start with describing it as a goodness of fit test rather than jumping straight into the null hypothesis, and yes, good that we're interested in a particular multinomial distribution.

The one thing i'd take out is the parenthetical bit about the default, and I'd add back the part about the `N` observations being from *a categorical distribution*. (I'd say the input is *one* observation from a multinomial distribution parametrized by total count `N` and the provided frequencies.)

Reply:
Edited - how's that?

Reply:
Committed!

Reply:
There is only one "multinomial distribution". The term is well-defined. If it is not what https://en.wikipedia.org/wiki/Multinomial_distribution describes, then it is not a multinomial distribution.